ddl/ddl_worker_test.go

```diff
@@ -137,6 +137,47 @@ func (s *testDDLSuite) TestTableError(c *C) {
 
 }
 
+func (s *testDDLSuite) TestViewError(c *C) {
+	store := testCreateStore(c, "test_view_error")
+	defer store.Close()
+
+	d := testNewDDL(context.Background(), nil, store, nil, nil, testLease)
+	defer d.Stop()
+	ctx := testNewContext(d)
+
+	// Schema ID is wrong, so dropping table is failed.
+	doDDLJobErr(c, -1, 1, model.ActionDropTable, nil, ctx, d)
+	// Table ID is wrong, so dropping table is failed.
+	dbInfo := testSchemaInfo(c, d, "test")
+	testCreateSchema(c, testNewContext(d), d, dbInfo)
+	job := doDDLJobErr(c, dbInfo.ID, -1, model.ActionDropTable, nil, ctx, d)
```

Comment:
ditto.

Reply:
@crazycs520 You advise only keep this check in `TestTableError`?

Reply:
ye.

Reply:
fine, How about now?